### ISSUE TITLE
508-defect-2 [FOCUS MANAGEMENT, SCREENREADER]: Focus on page load SHOULD be consistent

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#17004]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh Kim

## Details

This is similar to [this ticket for caregiver's post-submission page](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9305) @jenstrickland  identified earlier this year.

"On the final confirmation screen, the focus is on the h1, but had been on the h2 throughout the flow. On the confirmation screen the focus **should** be on the h2 "You've successfully submitted your application." This is of consequence for the screen reader experience."

## Tasks
- [ ] Check the comments section for other known tickets
- [ ] Consider the other forms we have confirmation pages for that might benefit from this work. 

## Acceptance Criteria

- [x] Set the focus to the page's `h2` confirmation.

## Environment

* Operating System: Mac, Windows
* Browser: Safari, Firefox
* Screenreading device: VoiceOver (Mac), NVDA (Firefox)
* Server destination: staging,  production

## Steps to Recreate

1. [Go to the form 28-8832 wizard page](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction).
2. Start screenreading device listed in Environment.
3. Complete the form and submit.
4. Confirm the focus goes to the `h1` instead of the `h2` confirmation.

## Solution

- [Complete this ticket first to fix the heading level order.](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16735)
- Set the focus to the page's `h2` confirmation. 

## WCAG or Vendor Guidance (optional)

* [W3C WCAG Consistent Identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html)

## Screenshots or Trace Logs

<img width="864" alt="Screen Shot 2020-12-07 at 12 21 47 PM" src="https://user-images.githubusercontent.com/14154792/101391892-fadb7880-3892-11eb-916f-9ed6f0e3d654.png">

**Comment by joshkimux** [comment]
@jason-gcio Yes absolutely. It's very much contextual-- would love to hop on a call with you Friday to talk it through.

**Comment by jason-gcio** [comment]
Do along side of: https://github.com/department-of-veterans-affairs/va.gov-team/issues/13019
